packages/ember-metal/lib/enumerable_utils.js

```diff
@@ -8,125 +8,183 @@ import {
 var splice = Array.prototype.splice;
 
 /**
- * Defines some convenience methods for working with Enumerables.
- * `Ember.EnumerableUtils` uses `Ember.ArrayPolyfills` when necessary.
- *
- * @class EnumerableUtils
- * @namespace Ember
- * @static
- * */
+  Defines some convenience methods for working with Enumerables.
+  `Ember.EnumerableUtils` uses `Ember.ArrayPolyfills` when necessary.
+
+  @class EnumerableUtils
+  @namespace Ember
+  @static
+ */
 
 /**
- * Calls the map function on the passed object with a specified callback. This
- * uses `Ember.ArrayPolyfill`'s-map method when necessary.
- *
- * @method map
- * @param {Object} obj The object that should be mapped
- * @param {Function} callback The callback to execute
- * @param {Object} thisArg Value to use as this when executing *callback*
- *
- * @return {Array} An array of mapped values.
+  Creates and returns a new array with the results of calling the specified
+  `callback` on each element in the enumerable. This method corresponds to
+  `map()` defined in JavaScript 1.6.
```

Comment:
By 1.6 here, you mean EcmaScript 2015? I'd rather see "ES6" or ES2015" referenced if anything.